x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -318,13 +329,16 @@ export const AlertsList: React.FunctionComponent = () => {
   );
 };
 
-function pickFromData(data: AlertTableItem[], ids: string[]): AlertTableItem[] {
-  const result: AlertTableItem[] = [];
-  for (const id of ids) {
-    const match = data.find(item => item.id === id);
-    if (match) {
-      result.push(match);
-    }
-  }
-  return result;
+function filterAlertsById(alerts: Alert[], ids: string[]): Alert[] {
+  return alerts.filter(alert => ids.includes(alert.id));
+}
+
+function convertAlertsToTableItems(alerts: Alert[], alertTypesIndex: AlertTypeIndex) {
+  return alerts.map(alert => ({
+    ...alert,
+    tagsText: alert.tags.join(', '),
+    alertType: alertTypesIndex[alert.alertTypeId]
+      ? alertTypesIndex[alert.alertTypeId].name
+      : alert.alertTypeId,
```

Comment:
total nit. but could be a little simpler: `alertTypesIndex[alert.alertTypeId]?.name ?? alert.alertTypeId`

Reply:
I didn't know about that, very cool! Will do the change